src/net/inv.rs

```diff
@@ -3118,6 +3118,7 @@ mod test {
             u64::max_value(),
             u64::max_value(),
             u32::max_value(),
+            u32::MAX,
```

Comment:
I was wondering about this, but I see `MAX` is now preferred. I think we can open an issue to update the others for consistency later.

Reply:
#3682